scripts/set-dm-password-by-email.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+"""
+    This script isn't really intended to be run on its own, instead being called from rotate-api-tokens.sh.
+    If you are going to run it on its own, it expects the following environment variables to be set:
+     - STAGE: preview, staging or production
+     - ACCOUNT_EMAIL: the email address of the DMP account whose password is to be changed
+     - ACCOUNT_PASSWORD: the new password for the DMP account
+
+    DMP_CREDENTIALS_REPO may also be needed to retrieve the required API tokens
+"""
+
+import sys
+
+sys.path.insert(0, '.')
+
+from os import environ
+from sys import exit
+
+from dmapiclient import DataAPIClient
+from dmscripts.helpers.auth_helpers import get_auth_token
+from dmutils.env_helpers import get_api_endpoint_from_stage
+
+if __name__ == "__main__":
```

Comment:
Would be super-nice to pull this `__main__` stuff into a function so we can move it & add tests later, but not a blocker

Reply:
Yeeeah I kinda wanted this script to not exist at all really - originally tried to include the source in the bash script as a one-liner.